src/main/java/org/opentripplanner/graph_builder/module/GtfsModule.java

```diff
@@ -181,6 +227,39 @@ private GtfsMutableRelationalDao loadBundle(GtfsBundle gtfsBundle)
 
         GtfsFeedId gtfsFeedId = gtfsBundle.getFeedId();
 
+        // check for duplicate bundle names and make sure that at most one of those bundles contains an auto-generated
+        // feed ID. If there is more than one, non-deterministic Feed IDs could be generated.
+        String bundleFilename = gtfsBundle.getPath().getName();
+        if (bundleFilenamesWithAutoGeneratedFeedId.contains(bundleFilename)) {
+            throw new RuntimeException(
+                String.format(
+                    "There are at least 2 bundles with the name `%s` that lack a feed_id value in a record in the feed_info.txt table. This will result in non-deterministic Feed ID generation!",
+                    bundleFilename
```

Comment:
The variable names and comments calling things `bundle` doesn't seem clear to me. I'd prefer `gtfsBundle*` or `gtfs*`. I would have no context for what a "bundle" is if I were just seeing this log statement with fresh eyes.